app/js/templates/adminTable.hbs

```diff
@@ -1,34 +1,124 @@
+                                </td>
+                     {{/unless}}
+                <tr>
+                </tr>
+                <tr>
+                    <td>
+                    {{#unless testNotTaken}}
+                           <button dataId="{{id}}" class="btn view-results-button btn_height">View Results</button>
+                     {{/unless}}
+                    </td>
+                </tr>
+            </table>
+        </td>
+        <td>
+{{!--          second column --}}
+            <table>
+                <tr>
+                    <th>Category Name:</th>
+                </tr>
+                <tr>
+                    <td>Score:</td>
+                </tr>
+                <tr>
+                    <td>Test Allocated:</td>
+                </tr>
+            </table>
+        </td>
+        <td>
+{{!--          third column --}}
+            <table>
+                <tr>
+                    <td class="categoryName">{{categoryName}}</td>
+                </tr>
+                <tr>
+                    <td class="score">{{score}}</td>
+
+
+                </tr>
+                <tr>
+                    <td class="testAllocated">{{testAllocated}}</td>
+                </tr>
+            </table>
+        </td>
+        <td>
+{{!--          forth column --}}
+            <table>
+                <tr>
+                    <td>Date:</td>
+                </tr>
+                <tr>
+                    <td>Time Allowed:</td>
+                </tr>
+                <tr>
+                    <td>Time Taken:</td>
+                </tr>
+
+            </table>
+        </td>
+        <td>
+{{!--          fifth column --}}
+            <table>
+                <tr>
+                    <td class="date">{{#if testNotTaken}}
+                                          {{testNotTaken}}
+                                      {{else}}
+                                           {{dateCreated}}
+                                      {{/if}}</td>
+                </tr>
+                <tr>
+                    <td class="time">{{timeAllowed}}</td>
+                </tr>
+                <tr>
+                    <td class="time">{{time}}</td>
+                </tr>
+            </table>
         </td>
-        <td class="name">{{name}}</td>
-        <td class="email">{{email}}</td>
-        <td class="categoryName">{{categoryName}}</td>
-        <td class="score">{{score}}</td>
-        <td class="percentage">{{percentage}}</td>
-        <td class="testAllocated">{{testAllocated}}</td>
-        <td class="time">{{timeAllowed}}</td>
-        <td class="time">{{time}}</td>
-        <td class="date">{{#if testNotTaken}}
-                              {{testNotTaken}}
-                         {{else}}
-                              {{dateCreated}}
-                         {{/if}}</td>
+</table>
 </tr>
 {{/each}}
```

Comment:
needs empty line at bottom of file to keep github happy

Reply:
have commited